mappings/net/minecraft/container/Slot.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS gp net/minecraft/container/Slot
```

Comment:
Class name already mapped `net/minecraft/container/slot/Slot

Though the fields `b` and `c` still need to be mapped